x-pack/plugins/fleet/public/constants/page_paths.ts

```diff
@@ -87,8 +88,16 @@ export const pagePathGetters: {
   base: () => [FLEET_BASE_PATH, '/'],
   overview: () => [FLEET_BASE_PATH, '/'],
   integrations: () => [INTEGRATIONS_BASE_PATH, '/'],
-  integrations_all: () => [INTEGRATIONS_BASE_PATH, '/browse'],
-  integrations_installed: () => [INTEGRATIONS_BASE_PATH, '/installed'],
+  integrations_all: ({ query, category }: { query?: string; category?: string }) => {
```

Comment:
nit: can we rename `query` to `searchTerm` to disambiguate from the URL query? Same on `integrations_installed`